src/views/components/application-status.njk

```diff
@@ -0,0 +1,48 @@
+<div class="govuk-grid-row govuk-!-margin-top-4">
+  <div class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-heading-l">Application status</h2>
+    <p class="govuk-body">Refresh the page to see updated information. If you exit this page, you can return to it by signing in to <a href="{{ Paths.ROOT_URI }}">the Apply to strike off and dissolve a company service</a>.</p>
+  </div>
+
+  {% set signatoryRows = [] %}
+  {% for signatory in signatories %}
+    {% set signatoryRows =
+      (
+        signatoryRows.push(
+          [
+            {
+              html: signatory.name if not signatory.on_behalf_name else '<div class="on_behalf_name_' + loop.index0 + '">' + signatory.on_behalf_name + '</div><div class="on_behalf_of_signatory_name_section_' + loop.index0 + '">signing on behalf of ' + signatory.name + '</div>',
```

Comment:
users can enter html for an on behalf name, we don't validate it. This will cause XSS if you use the html property. I don't think we need to because it's a single text string with no formatting.

Reply:
Good point. I think the GDS design system must prevent XSS. Tested if someone entered the on behalf name as `<script>alert("hello");</script>` and it didn't execute. But yeah we could do something like this:
```
{
    text: signatory.name if not signatory.on_behalf_name,
    html: '<div class="on_behalf_name_' + loop.index0 + '">' + signatory.on_behalf_name + '</div><div class="on_behalf_of_signatory_name_section_' + loop.index0 + '">signing on behalf of ' + signatory.name + '</div>' if signatory.on_behalf_name, 
    attributes: { id: "name-" + loop.index0 }
},
```

Reply:
Confirmed with Jayti that the on behalf name doesn't have to sit on a separate line from the other text, so no need for HTML anyway. By using nunjucks `text` prop it is auto encoded.